aegis-web/yo/app/scripts/controllers/extendedMetadataCtrl.js

```diff
@@ -39,7 +39,8 @@
 
 'use strict';
 
-
+const AEGIS_PROJECT_TEMPLATE_ID = 14;
```

Comment:
I'm still not sure if we should use only compliant JavaScript.

Reply:
`const` is already used in elsewhere `mainCtrl.js`. I tested with `grunt build` and got no JS errors in the browser. All browser versions since 2016 support `const` and `let`, see https://caniuse.com/#feat=const

But if prefered we can changed to `var`.